Police-Data-Accessibility-Project/data-source-map
# feature: add counties map

This PR adds a new map that highlights US counties from whose policing institutions PDAP has data.

## To test
1. Check out this branch, run `npm i` then `npm run dev`
2. Navigate to `localhost:8888/counties`

## Still to do
- [ ] Figure out mapbox data expressions for better handling of labels
- [ ] Atomize utils
- [ ] Update counties layer on color preference change (easy once utils are split out into multiple funcs)
- [ ] General cleanup

## Some ideas for iteration:
1. Do we want to look into any particular user interaction?
2. i.e.: do we want any sort of sidebar or popup here that displays data on a particular county when hovered or clicked?
3. Would it make sense to have a sort of legend included that would communicated to the user what the different opacity levels mean? (i.e. the population legend in [this chloropleth map](https://docs.mapbox.com/mapbox-gl-js/example/updating-choropleth/) example?)
